python/mlcroissant/README.md

```diff
@@ -127,6 +127,19 @@ class RecordSet(Node):
     )
 ```
 
+**Example 3**: specify a version:
```

Comment:
NIT I would probably say something about the fact that if none given it is V1.
And that, in some cases, it is important to select another version, like if a property is only available for a specific version, like below

Reply:
Done.

By default, all versions are supported, not just v1.

Reply:
PS: I also added documentation in python/mlcroissant/mlcroissant/_src/core/dataclasses.py (Metadata's docstring).